src/main/scala/com/cognite/sdk/scala/v1/resources/files.scala

```diff
@@ -212,17 +240,35 @@ object Files {
     deriveDecoder[FileDownloadLinkId]
   implicit val fileDownloadLinkExternalIdDecoder: Decoder[FileDownloadLinkExternalId] =
     deriveDecoder[FileDownloadLinkExternalId]
+  implicit val fileDownloadLinkInstanceId: Decoder[FileDownloadLinkInstanceId] =
```

Comment:
```suggestion
  implicit val fileDownloadLinkInstanceIdDecoder: Decoder[FileDownloadLinkInstanceId] =
```
for consistency with one above